debezium-server-voyager/debezium-server-voyagerexporter/src/main/java/io/debezium/server/ybexporter/EventQueue.java

```diff
@@ -281,6 +252,11 @@ public void warmUp(Map<Long, Long> totalEventsPerSegment) {
                 String line;
                 BufferedReader input;
                 try {
+                    // If the queue segment file has been deleted by the archiver, move to the next
+                    if (!Files.exists(Path.of(currentQueueSegmentPath))) {
+                        currentQueueSegmentIndex++;
```

Comment:
pls add a log that the segment is being skipped. because it has already been archived/deleted.

Reply:
Done